src/lib.rs

```diff
@@ -235,14 +239,7 @@ impl fmt::Display for ConstructionError {
     }
 }
 
-impl error::Error for ConstructionError {
-    fn description(&self) -> &str {
-        match *self {
-            ConstructionError::IDTooLarge => "can id too large",
-            ConstructionError::TooMuchData => "too much data",
-        }
-    }
-}
+impl error::Error for ConstructionError {}
```

Comment:
Same comment as above about this error cohabiting with the others in `src/err.rs`.